### PR TITLE
Granting and Deleting assets from Player Inventory

### PR DESCRIPTION
**Pull request by AlmightyMikkel**

### CODE REVIEW

File: Runtime/Game/LootLockerSDKManager.cs [hunk 1]
```diff
@@ -3524,6 +3556,24 @@ public static void OpenALootBoxForAssetInstances(int assetInstanceID, Action<Loo
             data.getRequests.Add(assetInstanceID.ToString());
             LootLockerAPIManager.OpenALootBox(data, onComplete);
         }
+
+        /// <summary>
+        /// Delete an Asset Instance from the current Player's Inventory.
+        /// </summary>
+        /// <param name="assetInstanceID">The asset instance ID of the you want to delete from the Players Inventory</param>
```
Comment (JohannesLoot) [inline Runtime/Game/LootLockerSDKManager.cs:3563]: Missing some words, suggesting to write;
"The asset instance ID of the asset instance you want to delete from the current Players Inventory"

File: Runtime/Game/Requests/AssetRequest.cs [hunk 2]
```diff
@@ -187,6 +194,17 @@ public class LootLockerActivateRentalAssetResponse : LootLockerResponse
...
         public int time_left { get; set; }
     }
+
+    public class LootLockerGrantAssetResponse : LootLockerResponse
+    {
+        public int id { get; set; }
+        public int asset_id { get; set; }
+        public int asset_variation_id { get; set; }
+        public int asset_rental_option_id { get; set; }
+        public string asset_ulid { get; set; }
```
Comment (JohannesLoot) [inline Runtime/Game/Requests/AssetRequest.cs:204]: Just want to double-check, is it ulid or uuid?
This https://ref.lootlocker.com/game-api/#asset
shows uuid and not ulid, but this response might be different.

**Review by JohannesLoot — DISMISSED** [review]
Looks great!
Solid work!
Left a small comment and a question :)

File: Runtime/Game/LootLockerSDKManager.cs [hunk 3]
```diff
@@ -3304,6 +3304,38 @@ public static void GetAssetsById(string[] assetIdsToRetrieve, Action<LootLockerA
             LootLockerAPIManager.GetAssetsById(data, onComplete);
         }
 
+        /// <summary>
+        /// Grant an Asset Instance to the Player's Inventory.
```
Comment (kirre-bylund) [inline Runtime/Game/LootLockerSDKManager.cs:3308]: I think "Instance" here is unnecessary.

File: Runtime/Game/LootLockerSDKManager.cs [hunk 4]
```diff
@@ -3304,6 +3304,38 @@ public static void GetAssetsById(string[] assetIdsToRetrieve, Action<LootLockerA
...
+        {
+            GrantAssetToPlayerInventory(assetID, null, null, onComplete);
+        }
+
+        /// <summary>
+        /// Grant an Asset Instance to the Player's Inventory.
+        /// </summary>
+        /// <param name="assetID">The Asset you want to create an Instance of and give to the current player</param>
+        /// <param name="assetVariationID">The id of the Asset Variation you want to grant</param>
+        /// <param name="assetRentalOptionID">the rental option id you want to give the Asset Instance</param>
```
Comment (kirre-bylund) [inline Runtime/Game/LootLockerSDKManager.cs:3321]:
```suggestion
        /// <param name="assetRentalOptionID">The rental option id you want to give the Asset Instance</param>
```

**Review by kirre-bylund — APPROVED** [review]
Agree with Johannes, beyond that it looks good.

File: Runtime/Game/Requests/AssetRequest.cs (hunk 2 above)
Comment (kirre-bylund) [reply Runtime/Game/Requests/AssetRequest.cs:204]: It is indeed asset_ulid :)

**Pull request merged**